scripts/gha/github.py

```diff
@@ -236,9 +236,9 @@ def list_workflows(token, workflow_id, branch):
   """https://docs.github.com/en/rest/reference/actions#list-workflow-runs-for-a-repository"""
   url = f'{GITHUB_API_URL}/actions/workflows/{workflow_id}/runs'
   headers = {'Accept': 'application/vnd.github.v3+json', 'Authorization': f'token {token}'}
-  data = {'event': 'workflow_dispatch', 'branch': branch}
+  data = {'event': 'workflow_dispatch', 'branch': branch, 'per_page': 5}
   with requests.get(url, headers=headers, data=json.dumps(data),
-                    stream=True, timeout=TIMEOUT) as response:
+                    stream=True, timeout=20) as response:
```

Comment:
Why the one-off change in the TIMEOUT value here?
Also, could you make it a different constant at the top of the file and then use that constant name here?

Reply:
Because we have timeout issue listing workflows.
https://github.com/firebase/firebase-cpp-sdk/actions/runs/3288289567/jobs/5426248412

Also created another constant.